index.js

```diff
@@ -249,7 +249,7 @@ Kareem.prototype.execPost = function(name, context, args, options, callback) {
         next();
       });
 
-      if (post.length === numArgs + 2) {
+      if (isErrorHandlingMiddleware(posts[currentPost], numArgs)) {
```

Comment:
same here:
previously it was just `post`, why now change to `posts[currentPost]` (while `post` still exists)

Reply:
Same as :point_up: